trackintel/preprocessing/positionfixes.py

```diff
@@ -413,7 +414,7 @@ def _generate_staypoints_sliding_user(
             # the duration of gap in the last two pfs
             gap_t = (pfs[curr]["tracked_at"] - pfs[curr - 1]["tracked_at"]).total_seconds()
 
-            # we want the spt to have long duration, but the gap of missing signal should not be too long
+            # we want the spt to have long duration, but the gap of two consecutive pfs should not be too long
```

Comment:
should we stick to the convention I have seen in other places for comments? like writing "positionfixes" in place of pfs, "staypoints" in place of stps etc..?

just wondering.. :)

Reply:
I am not sure if we have this conversion, maybe need to agree on one
But here the comments have been changed